v2/cmd/asoctl/internal/importing/importable_arm_resource.go

```diff
@@ -120,56 +154,31 @@ func (i *importableARMResource) Import(ctx context.Context, bar *mpb.Bar) ([]Imp
 	total := int64(len(childTypes) + 1)
 	bar.SetTotal(total, false)
 
-	bar.SetCurrent(1)
-
+	// While we're looking for subresources, we need to treat any errors that occur as independent.
+	// Some potential subresource types can have limited accessibility (e.g. the subscriber may not
+	// be onboarded to a preview API), so we don't want to fail the entire import if we can't import
+	// a single candidate subresource type.
+	var errs []error
 	for _, subType := range childTypes {
 		subResources, err := i.importChildResources(ctx, ref, subType)
+		//if subType == "Microsoft.ContainerService/managedClusters/trustedAccessRoleBindings" {
```

Comment:
Is this commented block intended to be left in here?

Reply:
No! Good catch.